bootstrap.py

```diff
@@ -383,7 +422,15 @@ def install_puppet_agent():
     call_yum("install", "puppet")
     enable_service("puppet")
 
-    puppet_conf = open('/etc/puppet/puppet.conf', 'wb')
+    puppet_major_version = get_puppet_version()
+    if puppet_major_version == 3:
+        puppet_conf_file = '/etc/puppet/puppet.conf'
+    elif puppet_major_version == 4:
```

Comment:
can we make this `puppet_major_version in [4,5]`? as Puppet5 uses the same path and that would make it more future proof

Reply:
👍